src/components/ButtonsToolbar/index.js

```diff
@@ -28,10 +28,10 @@ const ButtonsToolbar = ({
       <LiteralsProvider literals={literals}>
         <Toolbar modifiers={['mobile-reverse-order', 'bottom']}>
           <ToolbarList>
-            {onNavigatePrevPage && (
-              <UtrechtLinkButton onClick={onNavigatePrevPage} appearance="subtle-button">
+            {previousPage && (
+              <Link to={previousPage}>
```

Comment:
I think we still want to the `onClick` so that the parent component `NAVIGATE` is still dispatched, no?

Reply:
Ah I think I misunderstood your previous comment then, I thought you wanted me to remove it. I will add it again.